.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+	"presets": ["env"]
+}
```

Comment:
it might be confusing that this is in the repo root, while sheetify itself doesn't use a babel build step. i think it'd be better to pass this config in explicitly to the babelify transform in the test:

```js
.transform(babelify, { presets: ['env'] })
```

Reply:
Yeah, I didn't like that either, but I simply didn't know I could just pass it to the transform like this. So: will do.